### PR TITLE
OOP school library - Preserve data

### PR DESCRIPTION
**Pull request by elmar8287**
- Check the documentation for Ruby's JSON library.
- On exit from an app, it should save all the data (books, people, and rentals) into JSON files. One per type of data is stored and named accordingly (e.g. rentals.json).
- On startup of the app should be loaded all data from files.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
+require 'json'
+
+# rubocop:disable Metrics/ModuleLength
```
Comment (SajjadAhmad14) [inline app.rb:3]: Data should be stored in a separate JSON file. For example, rental data should be stored in `rentals.json`, and people data should be stored in `people.json`. Data is not being preserved anywhere when the user exits from the app.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi all, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git Flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.rb (hunk 1 above)
Comment (Donard97) [reply app.rb:3]: Thank you for letting us know where was the problem. We fixed everything and now data are stored in separate JSON files.

Comment (DeliceLydia) [reply app.rb:3]: Well addressed this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (Donard97) [reply app.rb:3]: @DeliceLydia  Thank you a lot!

**Pull request merged**